utils/getCountries.js

```diff
@@ -0,0 +1,32 @@
+const axios = require("axios");
+const comma = require("comma-number");
+
+const getCountries = async (spinner, table, countries) => {
+	if (countries) {
+		for (country of countries) {
+			const api = await axios.get(
+				`https://corona.lmao.ninja/countries/${country}`
+			);
+			if (api.data !== "Country not found") {
+				let data = Object.values(api.data);
+				data = data.map(d => comma(d));
+				table.push(data);
+			}
+		}
+		spinner.stopAndPersist();
+		console.log(table.toString());
+	}
+};
+
+const parseCountries = (input, separator = "-") => {
+	if (typeof input !== "string" || !input.includes(separator)) return false;
+
+	let countries = input.split("-").filter(country => country.length > 0);
```

Comment:
This should be `input.split(separator)` and not the hardcoded "-". To be corrected